README.md

```diff
@@ -1,57 +1,80 @@
+## Features
 
-## Getting Started
+This sample demonstrates the following **MSAL Node** concepts:
 
-### Prerequisites
+* Configuration
+* Login and logout
+* Acquiring an access token
+* Calling a web API
 
-(ideally very short, if any)
+## Contents
 
-- OS
-- Library version
-- ...
+| File/folder           | Description                                                  |
+|-----------------------|--------------------------------------------------------------|
+| `AppCreationScripts/` | Contains Powershell scripts for automating app registration. |
+| `App/authProvider.js` | Main authentication logic resides here.                      |
+| `App/main.js`         | Application main process.                                    |
+| `App/fetch.js`        | Axios HTTP client for calling endpoints with a bearer token. |
+| `App/renderer.js`     | Renderer processes and UI methods.                           |
+| `App/constants.js`    | Example user accounts in JSON                                |
+| `.env`                | Environment variables of authentication parameters.          |
 
-### Installation
+## Getting Started
 
-(ideally very short)
+### Prerequisites
 
-- npm install [package name]
-- mvn install
-- ...
+* [Node.js](https://nodejs.org/en/) must be installed to run this sample.
+* [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
 
-### Quickstart
-(Add steps to get up and running quickly)
+### Setup
 
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
+1. Register a [desktop app that calls web APIs](https://docs.microsoft.com/azure/active-directory/develop/scenario-desktop-app-registration) in the [Azure Portal](https://portal.azure.com).
+    * Navigate to the **Authentication** blade in the app registration.
+    * Click add a platform and choose **Mobile and Desktop application**
+    * For the **redirect URI**, enter `msal://redirect` into the input box.
+1. Clone this repository `git clone https://github.com/Azure-Samples/ms-identity-javascript-nodejs-desktop.git`
+1. Open the [.env](.env) file and provide the required configuration values.
+    1. Replace the string `Enter_the_Application_Id_Here` with your app/client ID on Azure AD portal.
+    1. Replace the string `Enter_the_Tenant_Id_Here` with your tenant ID on Azure AD portal.
+    1. Replace the string `Enter_the_Client_Secret_Here` with the client secret you created on Azure AD portal.
+    1. Replace the string `Enter_the_Cloud_Instance_Id_Here` with `https://login.microsoftonline.com/` (see **note** below).
+    1. Replace the string `Enter_the_Graph_Endpoint_Here`. with `https://graph.microsoft.com/` (see **note** below).
 
+> :information_source: *note*: This is for multi-tenant applications located on the global Azure cloud. For more information, see: [Use MSAL in a national cloud environment](https://docs.microsoft.com/azure/active-directory/develop/quickstart-v2-javascript-auth-code)
```

Comment:
>global Azure cloud [](start = 84, length = 18)

link to [https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud)